frontend/src/components/form/BookingsSearch.tsx

```diff
@@ -84,8 +107,16 @@ export default function Search({
                   color="primary"
                   size="small"
                   startIcon={<SaveIcon />}
-                  onClick={() => {
-                    updateUserLocation(values.locationId);
+                  onClick={async () => {
+                    try {
+                      await updateUserLocation(values.locationId);
+                      addAlert('Update User Default Location Successfully');
```

Comment:
I feel like the term "User" shouldn't be user facing, it's kind of impersonal. In this context, I'd just say "Your preferred location was saved"

Reply:
Thanks always, I'm very struggling with correct sentences.